source/slang/slang-ir.cpp

```diff
@@ -4382,6 +4391,7 @@ namespace Slang
 
         case kIROp_WitnessTable:
         case kIROp_StructType:
+        case kIROp_InterfaceType:
```

Comment:
I'm not 100% convinced that the way we are dumping IR interfaces here is the ideal, since they are currently *not* parent instructions (they have zero children, always), and they operate more like ordinary instructions at this point.

Of course, the dumping logic is not critical, so I'm fine with changing this up later.